ee/models/event_definition.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from posthog.models.event_definition import EventDefinition
+
+
+class EnterpriseEventDefinition(EventDefinition):
+    owner = models.ForeignKey("posthog.User", null=True, on_delete=models.PROTECT, related_name="event_definitions",)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
```

Comment:
`null=True` in a `CharField` is against best practices because `""` already represents an empty/nil state, `null` could be confusing.